src/main/java/unicash/model/util/SampleDataUtil.java

```diff
@@ -59,9 +79,9 @@ public static Transaction[] getSampleTransactions() {
                     new Name("Evening with friends"),
                     new Type("expense"),
                     new Amount(49.50),
-                    new DateTime("17-09-2023 00:00"),
+                    new DateTime("05-10-2023 00:00"),
```

Comment:
Just need to be careful about regressions with `GetTotalExpenditure` command unit tests since I rely on the months of the existing transactions